src/jit/Compiler.h

```diff
@@ -178,6 +180,11 @@ union InstructionValue {
     size_t targetLabelCount;
     // For immediate and local groups.
     Instruction* parent;
+    // For memory instructions.
+    struct {
+        uint64_t offset;
+        uint32_t memidx;
+    } memory;
```

Comment:
The offset can only be 32 bit as far as I know. Adding two members might increase this structure.

Reply:
Yes, it can only be 32 bits, it should be fixed now.

Though I am unsure how I could implement the offset, the pointer into the memory and the memory size (included in the newer version) without increasing the InstructionValue. It seemed the easiest approach.